README.md

```diff
@@ -223,6 +223,51 @@
   }
   ```
 
+  ### Generate Purchase form. ([Accept payment (Purchase)](https://wiki.wayforpay.com/en/view/852102))
+  
+  To generate a purchase button HTML you need to:
+  
+  ```ruby
+  Wayforpay::Payments.purchase_form(parameters)
+  ```
+
+  Required parameters:
+  
+| Parameter         | Description                                            |
+  |-------------------|--------------------------------------------------------|
+  | *orderReference*  | Uniq order id                                          |
+  | *orderDate*       | Order date in Uniq Timestamp format                    |
+  | *amount*          | Total order amount                                     |
+  | *currency*        | Order currency in ISO format (USD, UAH, CAD)           |
+  | *productName*     | An array of purchased products                         |
+  | *productCount*    | The number of each product in the purchase             |
+  | *productPrice*    | The each product price                                 |
+
+  An example of the call:
+  
+  ```erbruby
+    # checkout.html.erb
+    <%= Wayforpay::Helpers.purchase_form(
```

Comment:
Will it work as regular rails helper (`form_for`, `link_to`)

Reply:
nope. I can remove `self.` part from the method definition and in this case you can just include the module into your controller but in case if somebody is going to use the gem outside rails then it can be not so clear.
I prefer the current implementation because it's straight, explicit, and simple.

Reply:
@galulex should I do any changes?